addons/actions/src/manager.tsx

```diff
@@ -5,13 +5,10 @@ import { ADDON_ID, PANEL_ID } from '.';
 
 export function register() {
   addons.register(ADDON_ID, api => {
-    const channel = addons.getChannel();
```

Comment:
I removed this `const channel = addons.getChannel();` and its usage in ActionLogger props because  I didn't find any use of it in component codebase. 

ActionLogger Component is using [API to listen for channel event](https://github.com/storybooks/storybook/pull/5459/files#diff-7d97446ff6d37262ed00474530d5a79eR37) `api.on(EVENT_ID, this.addAction);`, and `action` get [its own ref to channel using `addons` ](https://github.com/storybooks/storybook/pull/5459/files#diff-e691237855e4c7244568777b789d0017R12)

Am I missing something?

Reply:
Perfect